assets/vue/components/PageBlocked.vue

```diff
@@ -107,6 +109,14 @@ export default {
     refreshData(data) {
       this.incidents = data.blocked;
     }
+  },
+  watch: {
+    groupFlavors: function (enabled) {
+      const url = new URL(location);
+      const params = url.searchParams;
+      enabled ? params.delete('group_flavors') : params.set('group_flavors', '0');
```

Comment:
Can't say i'm a fan of ternary use outside of expressions, but it's not a blocker.